### PR TITLE
test for CSS telephone capture

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
Test CCS telephone capture

# What has changed
New test

# How to test?
Install the caseapi: https://github.com/ONSdigital/census-rm-case-api/pull/99 and run the tests(s)

# Screenshots (if appropriate):

### CODE REVIEW

File: acceptance_tests/features/steps/ccs_property_listed.py [hunk 1]
```diff
@@ -93,7 +93,7 @@ def _field_callback(ch, method, _properties, body, context):
     ch.stop_consuming()
 
 
-def _create_ccs_property_listed_event(context, address_type="HH", address_level="U"):
+def _create_ccs_property_listed_event(context, address_type="HH", interview_required=True):
```
Comment (ryangrundy7) [inline acceptance_tests/features/steps/ccs_property_listed.py]: Address level needs to be there

**Comment by LeoKHoward** [comment]
I am getting failures for your test:

**Assertion Failed: Didn't find the expected number of messages**

**Review by NickGrantX — APPROVED** [review]
ATs passing for me

File: acceptance_tests/features/steps/ccs_property_listed.py [hunk 2]
```diff
@@ -129,3 +130,27 @@ def _create_ccs_property_listed_event(context, address_type="HH", interview_requ
...
+    context.messages_received = []
+    start_listening_to_rabbit_queue(Config.RABBITMQ_OUTBOUND_FIELD_QUEUE,
+                                    functools.partial(
+                                        _field_callback, context=context))
+
+    action_instruction = context.emitted_action_instruction
+    case_id = message['payload']['CCSProperty']['collectionCase']['id']
+    test_helper.assertEqual(case_id, action_instruction['caseId'])
+
+    response = requests.get(f'{Config.CASE_API_CASE_URL}{case_id}')
```
Comment (liamtoozer) [inline acceptance_tests/features/steps/ccs_property_listed.py]: Could use this [helper function](https://github.com/ONSdigital/census-rm-acceptance-tests/blob/5ead126b63d8c506abe857cdc2011d8dd2758eee/acceptance_tests/utilities/case_api_helper.py#L9)?

File: acceptance_tests/features/steps/telephone_capture.py [hunk 3]
```diff
@@ -76,3 +76,13 @@ def check_correct_individual_uac_updated_message_is_emitted(context):
 def telephone_capture_child_case_is_emitted(context):
     check_individual_child_case_is_emitted(context, context.telephone_capture_parent_case_id,
                                            context.individual_case_id)
+
+
+@step("there is a request for telephone capture against the new case")
+def telephone_capture_against_case(context):
```
Comment (liamtoozer) [inline acceptance_tests/features/steps/telephone_capture.py:82]: Minor comment but this is quite similar to [this function](https://github.com/ONSdigital/census-rm-acceptance-tests/blob/master/acceptance_tests/features/steps/telephone_capture.py#L19) (apart from the extra check). Might be a good time to pop this API call to `case_api_helper.py` so it's called a central place?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

**Pull request merged**